app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -183,6 +183,13 @@ def raw_metrics_for_counter(storage_conn, counter, start_time, end_time)
   end
 
   def get_counters(metrics_conn)
+    ems = target.ext_management_system
+
+    unless ems.supports_timeline?
```

Comment:
@djberg96 - I would probably move this to line 78 after we check for an EMS, this would keep the checks together and avoid calling methods if metrics isn't supported. Or is there a reason for placing this line here?